modules/intro/pages/index.adoc

```diff
@@ -46,17 +46,18 @@ For a more detailed description of the TigerGraph release process, please see th
 
 == Architecture overview
 The TigerGraph Platform runs on standard, commodity-grade Linux servers. 
-The core components (GSE and GPE) are implemented in C++ for optimal performance. TigerGraph system is designed to fit into your existing environment with a minimum of fuss.
+The core components of the Graph Storage Engine, or GSE, and the Graph Processing Engine, or GPE are implemented in C++ for optimal performance. TigerGraph is designed to fit into your existing environment with a minimum of fuss.
```

Comment:
1. If you want to use commas, then you need one more to fully bracket the explanatory phrase "or GPE",
changing
_the Graph Storage Engine, or GSE, and the Graph Processing Engine, or GPE are_
to
_the Graph Storage Engine, or GSE, and the Graph Processing Engine, or GPE, are_

2. I personally prefer to use parentheses when introducing an acronym. I feel it is more streamlined.
_the Graph Storage Engine (GSE) and the Graph Processing Engine (GPE) are_